db/dao/user.go

```diff
@@ -0,0 +1,9 @@
+package dao
+
+import "gorm.io/gorm"
+
+type User struct {
+	gorm.Model
+	Username string `json:"username" gorm:"unique;not null"`
+	Todos    []Todo `json:"todos" gorm:"foreignKey:UserID"`
```

Comment:
What is the purpose of mentioning `Todos` here? Which `UserID` is it referring to?